### DISCUSSION TITLE
Export to lightweight embed?

### DISCUSSION THREAD

**Discussion by jasonjmcghee** [motion-canvas/motion-canvas#976] [Q&A] [answered]
Hey there! I was curious if anyone has taken a crack at writing an export method that outputs a lightweight embed that could be used to include motion canvas animations in blog posts.

This isn’t the perfect example because it’s interactive, but uses p5.js- would be pretty cool to be able to do a similar format (but non interactive) with motion canvas.

https://jason.today/falling-sand

**Comment by aarthificial** [comment, marked as answer]
It's possible to [bundle the animation source code and run it on a website](https://github.com/motion-canvas/motion-canvas/issues/443) using our `motion-canvas-player` custom element.

Alternatively, there's an open issue for a Lottie exporter in our other repo: https://github.com/motion-canvas/exporters/issues/5
